src/cheri/insns/cbld_32bit.adoc

```diff
@@ -23,11 +23,13 @@ include::wavedrom/cbld.adoc[]
 Description::
 Copy `{cs2}` to `{cd}`.
 +
+Set `{cd}.ct=0` (that is, <<CBLD>> always builds _unsealed_ capabilities).
++
 Set `{cd}.tag=1` if:
 +
 . `{cs1}.tag` is set, and
 . `{cs1}` passes all <<section_cap_integrity,integrity>> checks, and
-. `{cs1}` is not sealed, and
+. `{cs1}` is not sealed (that is, `{cs1}.ct=0`), and
```

Comment:
I'm confused - what is cs1.ct if it's not cs1.tag?

Reply:
The capability type field.

Reply:
ok - it's not defined at such in the doc. I've been using {cs1}.tag which I thin is obvious but didn't know what this one was.

Reply:
`.ct` defines the sealed or unsealed bit, not whether it is tagged or not. Maybe one can use a different name, like `.otype`, because `.ct` is really hard to parse.